### DISCUSSION TITLE
posts/build-a-blog-using-hugo-papermod-github-pages/

### DISCUSSION THREAD

**Discussion by giscus** [sonnycalcr/sonnycalcr.github.io#1] [Announcements]
# posts/build-a-blog-using-hugo-papermod-github-pages/

前言 涉及工具 除了搭建博客和介绍如何使用这个博客框架这样一个流程本身，还包含了对 PaperMod 这个主题的一些细节的定制。 本文涉及到的工具或者网站： hugo papermod github postimage

https://sonnycalcr.github.io/posts/build-a-blog-using-hugo-papermod-github-pages/

**Comment by sonnycalcr** [comment]
这是第一个评论。

**Comment by sonnycalcr** [comment]
看一下代码和数学公式的渲染：

```html
<div>
$$
\boldsymbol{x}_{i+1}+\boldsymbol{x}_{i+2}=\boldsymbol{x}_{i+3}
$$
</div>
```

上面的数学公式想要正确地在  github 的 markdown 方言中渲染，得用这样的语法：

~~~
```math
\boldsymbol{x}_{i+1}+\boldsymbol{x}_{i+2}=\boldsymbol{x}_{i+3}
```
~~~

```math
\boldsymbol{x}_{i+1}+\boldsymbol{x}_{i+2}=\boldsymbol{x}_{i+3}
```

**Comment by Cooper0098** [comment]
up, 我想请问一下, 我现在搭好了网站, 不知道怎么才能把md文件插入的图片也传到网站上, 现在我md文件读取不到图片, 也没太搞懂怎么上传图片, 这个也是可以自动化流水式的吗?  我是用Typora来写md文档的🥲

**Reply by sonnycalcr** [reply]
我建议使用图床网站。我使用的是 postimage，网址是：https://postimages.org。这样一来，只需要用标准的 md 语法插入图片链接即可，其他不需要做任何事情。唯一的缺点可能是需要挂代理访问。不过，如果使用国内的图床，不稳定的风险可能会比较大。

**Reply by Cooper0098** [reply]
好的, 谢谢up😀

**Reply by Cooper0098** [reply]
up, 我误打误撞, 直接将图片上传到 posts 目录下自己创建的 img 文件夹，这样也可以在博客上显示这些图片。😁

**Reply by sonnycalcr** [reply]
可以的，无论哪种方法，能正常使用就成。

**Comment by P3lerA** [comment]
大佬，我这里mathjax渲染速度很慢，建议用官方文档里推荐的https://adityatelange.github.io/hugo-PaperMod/posts/math-typesetting/，在需要的笔记里开启，可以优化加载速度。
另外交一下作业:a1yce.com

**Reply by sonnycalcr** [reply]
Wow, awesome~

**Comment by lonebeetle** [comment]
我测试自己的评论链接的时候发现就卡在我点评论那个按钮那里了，不知道为什么诶

**Reply by lonebeetle** [reply]
我在hugo.yaml里补了几条看似不相关的设定，然后刷新了几次就可以了。谢谢up的分享！

**Comment by Minjie2003** [comment]
upup,这个图床的链接是flaticon的
